### PR TITLE
Miguel development

### PR DESCRIPTION
**Pull request by EcheverriaSoftware**
Finished Week 1 Walkthrough
Note: Will come back to style the layout better.

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -1,4 +1,5 @@
 <!DOCTYPE html>
+<!-- mvn package exec:java -->
```
Comment (aychin) [inline portfolio/src/main/webapp/index.html]: is this a leftover comment?

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -8,12 +9,46 @@
...
-      <p>This is my portfolio.</p>
-      <p>Click <a href="/hello">here</a> to see content created by server code.</p>
+      <h1>Miguel Echeverria</h1>
+      <p>My name is Miguel Echeverria and this is my first portfolio.</p>
+      <!--<p>Click <a href="/hello">here</a> to see content created by server code.</p>
       <p>Click here to get a random message:</p>
       <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <div id="greeting-container"></div>-->
+      <!--<img src="/images/download.jpg" />-->
```
Comment (aychin) [inline portfolio/src/main/webapp/index.html]: is this a leftover comment?

**Review by aychin — COMMENTED** [review]
Very nice!! I just made two notes for lines that look like leftover comments that should be cleaned up. Please make the changes as you see fit, resolve the comments, and re-request a review from the Conversation tab. Thanks!

**Comment by EcheverriaSoftware** [comment]
Removed all unnecessary comments from index.html

**Review by aychin — APPROVED** [review]

**Pull request merged**